mmcv/runner/hooks/logger/dvclive.py

```diff
@@ -11,48 +13,62 @@ class DvcliveLoggerHook(LoggerHook):
     It requires `dvclive`_ to be installed.
 
     Args:
-        path (str): Directory where dvclive will write TSV log files.
+        model_file (str):
+            Default None.
+            If not None, after each epoch the model will
+            be saved to {model_file}.
         interval (int): Logging interval (every k iterations).
             Default 10.
         ignore_last (bool): Ignore the log of last iterations in each epoch
             if less than `interval`.
             Default: True.
         reset_flag (bool): Whether to clear the output buffer after logging.
-            Default: True.
+            Default: False.
         by_epoch (bool): Whether EpochBasedRunner is used.
             Default: True.
+        kwargs:
+            Arguments for instantiating `Live`_
 
     .. _dvclive:
         https://dvc.org/doc/dvclive
+
+    .. _Live:
+        https://dvc.org/doc/dvclive/api-reference/live#parameters
     """
 
     def __init__(self,
-                 path,
```

Comment:
Does this mean that we have deprecated the argument `path`? If so, should we raise a warning and handle that?